### PR TITLE
Add first iteration of mobile docs

### PR DESCRIPTION
**Pull request by mironiasty**
This PR will add first iteration of mobile SDK documentation

### CODE REVIEW

File: docs/tutorial-react-native/connecting.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+create Room, add your app as Room's Participant and return token required to use that Room.
+
+To use that simply call `fetch`:
+```ts
+const response = await fetch(
+  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/*ROOM_NAME*/users/*USER_NAME*`,
+);
+
+const {url, token} = await response.json();
+```
```
Comment (MiloszFilimowski) [inline docs/tutorial-react-native/connecting.mdx:26]: **suggestion:** 
I think it's a bit too complicated and too much detail for a basic tutorial. Let's trim it down to simple steps. 


1. Get your account on [Fishjam Cloud](https://fishjam.io)
2. Create a Room with a User
```ts
const response = await fetch(
  `https://fishjam.io/api/v1/connect/*YOUR_TOKEN*/room-manager/*ROOM_NAME*/users/*USER_NAME*`,
);

const {url, token} = await response.json();
```

File: docs/tutorial-react-native/start-streaming.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 3
+---
+
+# Start streaming
+
+How to stream your camera
+
+:::tip Enable devices before connecting
```
Comment (MiloszFilimowski) [inline docs/tutorial-react-native/start-streaming.mdx:9]: **question:** What does it mean `Enable devices`? What devices?

Comment (MiloszFilimowski) [inline docs/tutorial-react-native/start-streaming.mdx:9]: **suggestion:** 
```suggestion
:::tip Enable camera and microphone before connecting
```

It's not really a tip but rather a requirement in this case 🤔

File: docs/tutorial-react-native/start-streaming.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+You can enable your camera and microphone before calling connect method. 
+This way, you can show user camera preview. Once connect method is called,
+enabled camera and microphone will start streaming to Room.
```
Comment (MiloszFilimowski) [inline docs/tutorial-react-native/start-streaming.mdx:13]: **suggestion:** 
```suggestion
You must enable your camera and microphone before calling the connect method if you want to stream video or audio. 
Once connected, the camera and microphone will start streaming to the Room.
```

File: docs/tutorial-react-native/connecting.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Connecting
+
+In order to connect, you just need to call `connect` method with data from previous step:
+
+```ts
+import { connect } from '@fishjam-cloud/react-native-client';
+
+
+await connect(url, token);
```
Comment (MiloszFilimowski) [inline docs/tutorial-react-native/connecting.mdx:36]: **suggestion:** 
I feel like we need more info here on what's achieved by calling `connect`. What happens and when should it be called? Maybe also a remark on how long the token lives.

File: docs/tutorial-react-native/connecting.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+## Disconnecting
+
+In order to close connection, you have to call `leaveRoom` method.
```
Comment (MiloszFilimowski) [inline docs/tutorial-react-native/connecting.mdx:41]: **suggestion:** 
example?

File: docs/tutorial-react-native/connecting.mdx (hunk 4 above)
Comment (MiloszFilimowski) [reply docs/tutorial-react-native/connecting.mdx:36]: I feel like we need more info here on what's achieved by calling `connect`. What happens and when should it be called? Maybe also a remark on how long the token lives.

After I connect, I'd like to see if it worked. Is there a place where I could check it on a web app/panel?

File: docs/tutorial-react-native/start-streaming.mdx (hunk 3 above)
Comment (mironiasty) [reply docs/tutorial-react-native/start-streaming.mdx:13]: Are you sure that you have to enable microphone/camera before connecting? 🤔 We need to verify it, as think you can connect without these devices

**Pull request merged**